tgui/packages/tgui-panel/styles/goon/chat-base.scss

```
@@ -331,7 +331,7 @@ em {
 
 /* SIZES */
 .large {
-  font-size: 3;
+  font-size: large;
```

ключевые слова тоже плохо, это не контролируемый рандом. Лучше

```suggestion
  font-size: 4em;
```
относительно и понятно

---

> Этот класс ранее не использовался, потому было незаметно.

нашел 3 упоминания в ``modules/admin/admin.dm``, не страшно

---

Мне другие классы переделать по подобию? Там `x-large`, `xx-large`

---

em как раз будет скейлится от текущего размера шрифта, а все эти large рандом и у юзера могут оказаться чем угодно.

Их стоит аннигилировать, это какое-то легаси времен бьенд чата. Но нужно скорее всего смотреть и подгонять новые размеры, просто заменив в классе можно что нибудь сломать. Тема отдельного ПР-а в общем